### ISSUE TITLE
severely broken community api listing concerns

### ISSUE THREAD

**Issue by shaneholloman** [BookStackApp/api-scripts#16]
Are the community repos tested and or endosed in any way before they are listed here in your github bookstack account

**Comment by cdrfun** [comment]
Hi @shaneholloman,
I think your question is answered by looking at the [README.md](https://github.com/BookStackApp/api-scripts/blob/main/README.md#bookstack-api-scripts):

> These scripts are not part an officially supported part of the BookStack project itself and therefore may be outdated or more likely to have bugs.

If there are any questions regarding specific scripts feel free to elaborate or to hand in a PR.

**Comment by ssddanbrown** [comment]
Yeah thanks @cdrfun. That does indeed cover the community scripts also.
Generally when a link to a community project is added I'll go a quick look over to check there's some substance there, it's under an open source license, and that it's BookStack specific, but I don't specifically run and/or test each project.
